app/build.gradle

```diff
@@ -62,4 +62,12 @@ dependencies {
     compile 'com.android.support:design:25.3.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
     testCompile 'junit:junit:4.12'
+    // parse
+    compile 'com.parse.bolts:bolts-android:1.4.0'
+    compile 'com.parse:parse-android:1.14.1'
+    compile 'com.parse:parseinterceptors:0.0.2'
+    compile 'com.parse:parsefacebookutils-v4-android:1.10.4@aar'
+    // butterknife
+    compile 'com.jakewharton:butterknife:8.4.0'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.4.0'
```

Comment:
@ankitj lets not use better knife and start using DataBinding as that's more official way for doing the same thing